integration_tests/testsuite/deploy_app.py

```diff
@@ -43,7 +51,26 @@ def _set_builder_image(builder):
                 fout.write(line.replace('${STAGING_BUILDER_IMAGE}', builder))
 
 
-def deploy_app(base_image, builder_image, appdir, yaml):
+def _record_latency_to_bigquery(deploy_latency, language):
+    current_date = datetime.datetime.now()
+    row = [(language, current_date, deploy_latency)]
+
+    project = os.environ.get(DEPLOY_LATENCY_PROJECT_ENV)
+    client = bigquery.Client(project=project)
```

Comment:
We'll need to give the service account we use in our Kokoro jobs access to this project. Otherwise, I think this client creation will fail.